rotkehlchen/globaldb/schema.py

```diff
@@ -222,22 +222,22 @@
 
 DB_CREATE_CUSTOM_ASSET = """
 CREATE TABLE IF NOT EXISTS custom_assets(
-    identifier INTEGER NOT NULL PRIMARY KEY,
+    identifier TEXT NOT NULL PRIMARY KEY,
     name TEXT NOT NULL,
     symbol TEXT,
     notes TEXT,
-    type TEXT
+    type TEXT,
+    FOREIGN KEY(identifier) REFERENCES assets(identifier) ON UPDATE CASCADE ON DELETE CASCADE
 );
 """
 
-DB_CREATE_ASSET_COLLECTION_PROPERTIES = """
-CREATE TABLE IF NOT EXISTS asset_collection_properties(
-    identifier TEXT NOT NULL PRIMARY KEY,
+DB_CREATE_ASSET_COLLECTIONS = """
+CREATE TABLE IF NOT EXISTS asset_collections(
```

Comment:
how is this table supposed to be used? What is it? It has no connection to any other table.